components/service/glean/src/main/java/mozilla/components/service/glean/net/ConceptFetchHttpUploader.kt

```diff
@@ -33,6 +35,16 @@ class ConceptFetchHttpUploader(
         const val DEFAULT_CONNECTION_TIMEOUT = 10000L
         // The timeout, in milliseconds, to use when reading from the server.
         const val DEFAULT_READ_TIMEOUT = 30000L
+
+        /**
+         * Export a constructor that is usable from Java.
+         *
+         * This looses the lazyness of creating the `client`.
+         */
+        @JvmStatic
+        fun fromClient(client: Client): ConceptFetchHttpUploader {
```

Comment:
Firefox Reality would need to use a different uploader. Do we need to do this for other uploaders as well? @pocmo

Reply:
I'm not sure I fully understand. The 1:1 migration for Firefox Reality would be to continue using `HttpUrlConnectionClient` (https://github.com/MozillaReality/FirefoxReality/issues/3339)?

Reply:
`Client` is coming from concept-fetch.
So as long as whatever they want to use adheres to that they're fine.